platforms/posix/cmake/sitl_target.cmake

```diff
@@ -64,7 +64,8 @@ set(models none shell
 	if750a iris iris_opt_flow iris_opt_flow_mockup iris_vision iris_rplidar iris_irlock iris_obs_avoid iris_rtps solo typhoon_h480
 	plane
 	standard_vtol tailsitter tiltrotor
-	hippocampus rover)
+	hippocampus rover
+	uuv_hippocampus)
```

Comment:
What is the difference between the hippocampus and the uuv_hippocampus?

Reply:
hippocampus is the old implementation (which is still based on the MC-airframe) which is still used in the tutorial with an example app.

Reply:
@hamishwillee Would it make sense to deprecate the previous tutorial and switch to the new one? I believe the previous module was not really well maintianed

Reply:
@Jaeyoung-Lim  @DanielDuecker 

I would completely delete the old support. The fact that you've "done this right" in this version is awesome, and what we want people to learn from.

Daniel, as far as I know the only docs for this is the launch command in http://dev.px4.io/master/en/simulation/gazebo.html#uuv

1. Can you update that with the new launch command?

It would be great to have some documentation to help others extend this work. Our docs currently make it pretty easy to set up an MC, VTOL and FW. However it is hard to set up a rover, boat or helicopter because there isn't much supporting material. I'd hate for that to be the case for this.

Generally we need "whatever you think would make it possible for someone to learn from and extend your work". Some ideas:
1. Dev: Explain the mixer and how it might be extended: http://dev.px4.io/master/en/concept/mixing.html
1. Dev: If there is a geometry file, explain how that works.
1. Explain the controller. E.g. like these diagrams: http://dev.px4.io/master/en/flight_stack/controller_diagrams.html
1. User: Explain any general considerations for assembly: http://docs.px4.io/master/en/assembly/
1. User: Explain how to set up a real vehicle http://docs.px4.io/master/en/airframes/
1. Explain special configuration: http://docs.px4.io/master/en/config/

Does that make sense?

Reply:
okay, I will remove the the old uuv-files.
@hamishwillee are you referring to the stil_gazebo part with "geometry file"?

We have quite some docs (so far internal) for assembly etc, I will see which parts would be helpful to others and add the to the docs.

It absolutely makes sense to pave to way to others who are interested in extending the framework with their own ideas.

Reply:
Hi Daniel,

Re geometry file I am referring to an undocumented part of the definition of PX4. Unfortunately I can't say anything else because I haven't ever been able to get any useful information on it. I was being opportunistic - and hoping you might :-)

> We have quite some docs (so far internal) for assembly etc, I will see which parts would be helpful to others and add the to the docs.

Thank you. That will be excellent!